### PR TITLE
Add Farsi language to i18n

### PR DESCRIPTION
**Pull request by Hasan-Mir**

### CODE REVIEW

**Comment by fabian-hiller** [comment]
Thank you for this PR! Do you know another native speaking developer that could review the translations for me?

**Comment by Hasan-Mir** [comment]
@ImMo3n

**Comment by fabian-hiller** [comment]
@ImMo3n are you able to review it?

**Comment by ImMo3n** [comment]
@fabian-hiller I've looked at it, looks good.

**Comment by fabian-hiller** [comment]
Thank you! I plan to merge it soon!

**Comment by fabian-hiller** [comment]
I have added translations for some new actions we have recently added. Can you check if my translations are correct?

File: packages/i18n/src/fa.ts [hunk 1]
```diff
@@ -5,17 +5,21 @@ const language: Language = {
...
+    base64:         (issue) => `Base64 نامعتبر: ${issue.received} دریافت شد`,
     bic:            (issue) => `BIC نامعتبر: ${issue.received} دریافت شد`,
     bytes:          (issue) => `بایت‌های نامعتبر: ${issue.expected} انتظار می‌رفت اما ${issue.received} دریافت شد`,
     check:          (issue) => `ورودی نامعتبر: ${issue.received} دریافت شد`,
     checkAsync:     (issue) => `ورودی نامعتبر: ${issue.received} دریافت شد`,
+    checkItems:     (issue) => `ورودی نامعتبر: ${issue.received} دریافت شد`,
     creditCard:     (issue) => `کارت اعتباری نامعتبر: ${issue.received} دریافت شد`,
     cuid2:          (issue) => `Cuid2 نامعتبر: ${issue.received} دریافت شد`,
     decimal:        (issue) => `اعشار نامعتبر: ${issue.received} دریافت شد`,
+    digits:         (issue) => `رقم نامعتبر: ${issue.received} دریافت شد`,
```
Comment (Hasan-Mir) [inline packages/i18n/src/fa.ts:17]:
```ts
digits:         (issue) => `ارقام نامعتبر: ${issue.received} دریافت شد`,
```

File: packages/i18n/src/fa.ts [hunk 2]
```diff
@@ -5,17 +5,21 @@ const language: Language = {
   code:             'fa',
   schema:           (issue) => `نوع نامعتبر: ${issue.expected} انتظار می‌رفت اما ${issue.received} دریافت شد`,
   specific: {
+    base64:         (issue) => `Base64 نامعتبر: ${issue.received} دریافت شد`,
     bic:            (issue) => `BIC نامعتبر: ${issue.received} دریافت شد`,
     bytes:          (issue) => `بایت‌های نامعتبر: ${issue.expected} انتظار می‌رفت اما ${issue.received} دریافت شد`,
     check:          (issue) => `ورودی نامعتبر: ${issue.received} دریافت شد`,
     checkAsync:     (issue) => `ورودی نامعتبر: ${issue.received} دریافت شد`,
+    checkItems:     (issue) => `ورودی نامعتبر: ${issue.received} دریافت شد`,
```
Comment (Hasan-Mir) [inline packages/i18n/src/fa.ts:13]:
```ts
checkItems:     (issue) => `آیتم نامعتبر: ${issue.received} دریافت شد`,
```

**Comment by Hasan-Mir** [comment]
Now it's ok.

**Comment by mammadataei** [comment]
Is there any plan to merge this PR? Is there anything I can help with?

**Comment by fabian-hiller** [comment]
Yes, I will probably merge it in the next days.

**Comment by fabian-hiller** [comment]
v1.0.0-beta.1 of i18n is available

**Pull request merged**